packages/nimble-components/src/wafer-map/specs/features/rendering.md

```diff
@@ -82,6 +82,48 @@ The POC is found in this branch [Worker Rendering POC](https://github.com/ni/nim
 
 ### Data Structure and Interface
 
+The best solution to solve the API of the wafermap is to use both of the proposed methods.
+
+The Public API will be the following:
+
+```TS
+public diesTable: Table<{
+        colIndex: Int32,
+        rowIndex: Int32,
+        value: Float32,
+        tags: Uint32;
+        metadata: never;
+    }>
+```
+
+This will be the [Apache Arrow](https://arrow.apache.org/docs/js/classes/Arrow_dom.Table.html) table schema.
+The row and column indices will be `Int32` columns, the values will be `Float32` columns.
+The tags for each die will be represented as a 32 bit mask stored in a `Uint32` column.
+The metadata column will be stored in an wildcard typed column.
+
+This approach has the benefits of a row based format that aligns well with the existing public api, a nice public API to use and the ease of future improvements.
```

Comment:
```suggestion
This approach has the benefits of a row based format that aligns well with the existing public API, as well as a nice public API that easily allows future improvements.
```
I'm not exactly sure if this rewording is what you were trying to get across, but I think it reads better.